src/display/views/threads/components/CheckboxTable.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import checkboxHOC from 'react-table/lib/hoc/selectTable';
 import ReactTable from 'react-table';
+import colors from '../../../../infrastructure/constants/colors';
```

Comment:
Something developer-facing I'd like to look at is using path mapping to get rid of the `../../`. We should be able to configure jest and webpack to resolve things from src/ given some token, so e.g.
```diff
-import colors from '../../../../infrastructure/constants/colors';
+import colors from '~/infrastructure/constants/colors';
```
What do you think?

Reply:
I like this very much. If it's something you're familiar with the settings for and want to submit a PR, feel free; otherwise I'll look into getting this updated in a week or two when my schedule eases up. Good suggestion!